### PR TITLE
Ported NFBProcessor class

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #17 partly.

**Changes**:
- Ported the NFBProcessor class from the neurolab-desktop to neurolab-android. The class has immediate usages in all the four program modes.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@CloudyPadmal @mariobehling @iamareebjamal Could you please review the PR.

File: app/src/main/java/io/neurolab/model/NFBProcessor.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package io.neurolab.model;
+
+import io.neurolab.settings.FeedbackSettings;
+
+public class NFBProcessor implements Task
+{
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/model/NFBProcessor.java]: Follow Java style guideline. not C++

**Comment by suresh-gandhi** [comment]
@iamareebjamal Did the changes. Please review and merge. Let me know if anything else needs to be changed.

**Comment by suresh-gandhi** [comment]
@mariobehling @CloudyPadmal @iamareebjamal Could you please review and merge. Let me know if something needs to be changed.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**